x-pack/legacy/plugins/lens/public/xy_visualization_plugin/xy_expression.tsx

```diff
@@ -26,7 +26,13 @@ import {
 import { EuiIcon, EuiText, IconType, EuiSpacer } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
 import { i18n } from '@kbn/i18n';
+import { npStart } from 'ui/new_platform';
```

Comment:
Definitely need to remove this

Reply:
Side note: I cleaned up the shimmed plugin structure in https://github.com/elastic/kibana/pull/56976